### DISCUSSION TITLE
Browser Extension to Mirror Full Functionality of Ente Auth App

### DISCUSSION THREAD

**Discussion by asmsayem04** [ente-io/ente#2060] [Enhancements]
Ente Auth is renowned for being a comprehensive open-source two-factor authentication app. Kudos to the developers for their exceptional work. It would be immensely beneficial if a browser extension were available that encompasses all the functionalities of the native app, in addition to the web interface, enhancing user experience and accessibility.

**Comment by GalaxyXYZ888** [comment]
Yeah the web interface was not what I have been looking for.. A browser extension with autocomplete would be ideal

**Comment by vishnukvmd** [comment]
Closing in favor of https://github.com/ente-io/ente/discussions/1041.
